opentelemetry-jaeger/src/lib.rs

```diff
@@ -312,11 +312,11 @@ impl PipelineBuilder {
     }
 
     /// Build a configured `sdk::Provider` with the recommended defaults.
```

Comment:
Should probably change this as well 
```suggestion
    /// Build a configured `sdk::TraceProvider` with the recommended defaults.
```